apps/frontpage/components/community/events.tsx

```diff
@@ -33,9 +30,27 @@ export const Events: FC = () => {
           </div>
         </div>
         <div className="flex flex-1 gap-4 rounded border border-zinc-300 p-6 md:gap-6 md:p-8 dark:border-slate-700">
-          <TwitterIcon className="h-10 w-10 text-[#4999e9]" />
+          <DiscordIcon className="h-10 w-10 text-[#5a65ea]" />
+          <div className="flex-1">
+            <h2 className="text-lg font-bold">Follow #announcements chat</h2>
+            <p className="mb-3 text-zinc-500">
+              Join our community chat to learn about live events and streams.
+            </p>
+            <a
+              className="flex items-center gap-2 text-blue-500"
+              href="https://discord.gg/storybook"
+            >
+              Chat now
+              <ChevronSmallRightIcon />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div className="mb-8 flex flex-col gap-8 md:flex-row">
+        <div className="flex flex-1 gap-4 rounded border border-zinc-300 p-6 md:gap-6 md:p-8 dark:border-slate-700">
+          <BlueSkyIcon className="h-10 w-10" color="#0085ff" />
           <div className="flex-1">
-            <h2 className="text-lg font-bold">Twitter</h2>
+            <h2 className="text-lg font-bold">Follow on BlueSky</h2>
```

Comment:
This section has the Twitter URL intead of the Bluesky URL.

https://bsky.app/profile/storybook.js.org